repository/src/main/java/com/epam/esm/repository/OrderRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.esm.repository;
+
+import com.epam.esm.entity.Order;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface OrderRepository {
```

Comment:
You don't need to add annotations @Repository on interfaces

Reply:
fixed and removed